### ISSUE TITLE
Bot doesn't work with node v14.15.0

### ISSUE THREAD

**Issue by lenisko** [5310/discord-bot-castform#15]
Hey!

Looks like bot "stopped" cycle job with node v14.15.0.
So even when chosen minute hits it does nothing.

I have same config on other machine with node v12.19.0 and it's working just fine.

```
$ npm start --verbose
npm info it worked if it ends with ok
npm verb cli [ '/usr/bin/node', '/usr/bin/npm', 'start', '--verbose' ]
npm info using npm@6.14.8
npm info using node@v14.15.0
npm verb run-script [ 'prestart', 'start', 'poststart' ]
npm info lifecycle discord-bot-castform@2.0.3~prestart: discord-bot-castform@2.0.3
npm info lifecycle discord-bot-castform@2.0.3~start: discord-bot-castform@2.0.3

> discord-bot-castform@2.0.3 start /home/discord/library/discord-bot-castform
> node index.js

Keepalive server is running
Castform is running
```

**Comment by 5310** [comment]
Can't really reproduce this @lenisko. 

Here's what I did:
1. Checked Node: `node -v` is `v14.15.0`.
1. Pulled the v2-webhook branch, which is the latest.
2. Pasted in one of the regions for our server. I hope this isn't changing anything.
3. Edited [line 159](https://github.com/5310/discord-bot-castform/blob/v2-webhook/index.js#L159) to change to a close-by minute for testing.
4. Ran `npm start --verbose` and waited for it to trigger.

It works as expected. I even got a push into our server.
